apps/backoffice-v2/src/pages/BusinessesAlerts/BusinessesAlerts.page.tsx

```diff
@@ -0,0 +1,55 @@
+import { BusinessAlertsTable } from '@/pages/BusinessesAlerts/components/BusinessAlertsTable';
+import { useBusinessAlertsLogic } from '@/pages/BusinessesAlerts/hooks/useBusinessAlertsLogic/useBusinessAlertsLogic';
+import { AlertsHeader } from '@/pages/TransactionMonitoringAlerts/components/AlertsHeader';
+import { AlertsPagination } from '@/pages/TransactionMonitoringAlerts/components/AlertsPagination/AlertsPagination';
+import { NoAlerts } from '@/pages/TransactionMonitoringAlerts/components/NoAlerts/NoAlerts';
+import { isNonEmptyArray } from '@ballerine/common';
+import { Outlet } from 'react-router-dom';
+
+export const BusinessesAlerts = () => {
+  const {
+    alerts,
+    isLoadingAlerts,
+    assignees,
+    labels,
+    authenticatedUser,
+    page,
+    onPrevPage,
+    onNextPage,
+    onPaginate,
+    isLastPage,
+    search,
+    onSearch,
+  } = useBusinessAlertsLogic();
+
+  return (
+    <div className="flex h-screen flex-col px-6 pb-6 pt-10">
+      <h1 className="pb-5 text-2xl font-bold">Businesses Ongoing Monitoring</h1>
+      <div className="flex h-full flex-col gap-6 overflow-auto">
+        <div>
+          <AlertsHeader
+            assignees={assignees ?? []}
+            labels={labels ?? []}
+            authenticatedUser={authenticatedUser}
+            search={search}
+            onSearch={onSearch}
+          />
```

Comment:
If this changes for alerts it would affect business alerts, I think a copy-paste here would be safer.